xbmc/addons/AddonManager.h

```diff
@@ -69,6 +69,12 @@ namespace ADDON
     bool Init();
     void DeInit();
 
+    CAddonMgr();
```

Comment:
If we want to avoid people creating new instances of `CAddonMgr` or creating copies of it we could make `CServiceMgr` a friend class and make these methods (apart from the destructor) protected.